showcase/config/json-server/data.json

```diff
@@ -3,5 +3,20 @@
     { "uuid": "1", "username": "janedoe", "firstName": "Jane", "lastName": "Doe", "language": "en", "roles": ["admin"] },
     { "uuid": "2", "username": "johndoe", "firstName": "John", "lastName": "Doe", "language": "fr", "roles": ["manager"] },
     { "uuid": "3", "username": "chucknorris", "firstName": "Chuck", "lastName": "Norris", "language": "nl", "roles": ["developer"] }
-  ]
+  ],
+  "userprofile": [
+    {
+      "uuid": "4cc28d14-16d3-46e7-a05a-487533c4cc26",
+      "username": "doejohn",
```

Comment:
In the defined users above, the username is `johndoe`. I think we should change this to `johndoe` too

Reply:
Good catch!... sure, I'll fix it ;)